### ISSUE TITLE
Crash on interacting with bark stripper on the ground storage (e.g. plank pile)

### ISSUE THREAD

**Issue by Craluminum2413** [TaskaRaine/Ancient-Tools#19]
```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.16.0 (Stable)01/16/2022 04:21:21: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at AncientTools.Items.ItemAdze.OnHeldInteractStart(ItemSlot slot, EntityAgent byEntity, BlockSelection blockSel, EntitySelection entitySel, Boolean firstEvent, EnumHandHandling& handling) in C:\Projects\vsmods\mods\ancienttools\src\items\ItemAdze.cs:line 57
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseActiveSlotItem(BlockSelection blockSel, EntitySelection entitySel, EnumHandInteract useType, EnumHandHandling& handling)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseActiveSlotItem(BlockSelection blockSel, EntitySelection entitySel)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   at _lb8GGBWOchQLOU3RFFw58Ijkh1D._1NlBQQ8qS5idQptIBQaAbARuOfxA(Single )
   at _vFhcDU5n6Z0hUte90jdaxlrDjR5._HuDdfzha8ZKZvU2y0RbH58r7VzR(Single )
   at _vFhcDU5n6Z0hUte90jdaxlrDjR5._XZOFA2JA54yZjH15yys9W7vL9hj(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 476
   at OpenTK.GameWindow.DispatchRenderFrame() in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 452
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 375
   at _FKpOpQGqqf3cJWdQyWh1ysxsDBb._pYad7oBcQ5qtzQkYGDu4NFHQxhK(_5x3PJgrln4IBXPDNSNN0fecxKFb , String[] )
   at _sejP0Tcz8wTg70ptIdRr3CwQmrE._pYad7oBcQ5qtzQkYGDu4NFHQxhK(ThreadStart )
```

**Comment by TaskaRaine** [comment]
Seems like Attributes are null instead of empty on blocks without any attributes. It'll be fixed for the next update.

**Comment by TaskaRaine** [comment]
This should be fixed now in version 1.3.3. Thanks so much for the report!
